### ISSUE TITLE
Bad startup message that misses version

### ISSUE THREAD

**Issue by mrdon** [launchdarkly/ld-daemon#2]
For 1.0.1, I see this in the logs:

Starting LaunchDarkly daemon version DEV with configuration file /etc/ld-daemon.conf
Initialized stream

**Comment by jkodumal** [comment]
Interesting. The way that's supposed to work is that `goxc` uses a linker variable at build time to replace the version variable with the git SHA. I'll take a closer look.

**Comment by jkodumal** [comment]
Looks like the root cause is that we're passing in `Version` as the linker variable, but the variable in the source is called `VERSION`. Also, we're not actually passing in the release version, we're passing in the build version. Historically, this is because the build template we used for ld-daemon is what we set up for our microservices, where we just use SHAs for versions.

I'm going to try to set things up to get both-- so you'll see

"Starting LaunchDarkly daemon version 1.0.1 (build ae41bf) with configuration file /etc/ld-daemon.conf"

when I'm done.
